### PR TITLE
Database setup

### PR DESCRIPTION
**Pull request by allieIsOkay**
Adds database and model validations

### CODE REVIEW

**Review by teresa-m-knowles — APPROVED** [review]
We did this together, it all looks good to me.

File: db/migrate/20190209184525_update_order.rb [hunk 1]
```diff
@@ -0,0 +1,5 @@
+class UpdateOrder < ActiveRecord::Migration[5.1]
+  def change
+    change_column_default :orders, :status, "pending"
```
Comment (timnallen) [inline db/migrate/20190209184525_update_order.rb:3]: Nice job thinking of default values for the statuses of things!

**Review by timnallen — COMMENTED** [review]
Looks great.

**Review by csvlewis — APPROVED** [review]
Good work, our database should be all set up now!

**Pull request merged**